t/t2260-job-list.t

```diff
@@ -2216,18 +2216,7 @@ test_expect_success 'job-list returns expected jobspec changes after reload' '
 test_expect_success 'job stats in each queue correct after reload' '
 	defaultq=`flux job stats | jq ".queues[] | select( .name == \"defaultqueue\" )"` &&
 	updateq=`flux job stats | jq ".queues[] | select( .name == \"updatequeue\" )"` &&
-	echo $defaultq | jq -e ".job_states.depend == 0" &&
-	echo $defaultq | jq -e ".job_states.priority == 0" &&
-	echo $defaultq | jq -e ".job_states.sched == 0" &&
-	echo $defaultq | jq -e ".job_states.run == 0" &&
-	echo $defaultq | jq -e ".job_states.cleanup == 0" &&
-	echo $defaultq | jq -e ".job_states.inactive == 0" &&
-	echo $defaultq | jq -e ".job_states.total == 0" &&
-	echo $defaultq | jq -e ".successful == 0" &&
-	echo $defaultq | jq -e ".failed == 0" &&
-	echo $defaultq | jq -e ".canceled == 0" &&
-	echo $defaultq | jq -e ".timeout == 0" &&
-	echo $defaultq | jq -e ".inactive_purged == 0" &&
```

Comment:
doh!  

Minor recommendation, comment here and possibly in commit message, a job is submitted to the "defaultqueue" but an update plugin converts it a different queue.  Therefore on reload, no stats exist for "defaultqueue", thus it is empty.

Reply:
I added a comment

Reply:
I added a comment, thanks!